src/g_nextweapon.c

```diff
@@ -138,12 +138,12 @@ typedef enum
 {
     nw_state_none,
     nw_state_activate,
-    nw_state_currently_active,
     nw_state_deactivate,
     nw_state_cmd
 } next_weapon_state_t;
 
 static next_weapon_state_t state;
+static boolean currently_active;
```

Comment:
Should this be reset to `false` in `G_NextWeaponReset`?

Reply:
Most probably yes!